bot/models/levels.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional
+
+from .model import Model
+
+
+class Levels(Model):
+    rank: Optional[int]
+    guild_id: int
+    user_id: int
+    old_total_xp: Optional[int]
+    total_xp: int
+
+    @classmethod
+    async def update(cls, guild_id: int, user_id: int, total_xp: int):
+        query = """INSERT INTO levels (guild_id, user_id, total_xp)
+                    VALUES ($1, $2, $3)
+                    ON CONFLICT (guild_id, user_id)
+                    DO UPDATE
+                        SET total_xp = levels.total_xp + $3, last_msg = CURRENT_TIMESTAMP
+                    WHERE
+                        levels.guild_id = $1
+                        AND levels.user_id = $2
+                        AND EXTRACT(EPOCH FROM (NOW() - levels.last_msg)) > 60
+                    RETURNING total_xp - $3 AS old_total_xp, total_xp, guild_id, user_id;
+"""
```

Comment:
```suggestion
    @classmethod
    async def update(cls, guild_id: int, user_id: int, total_xp: int):
        query = """
            INSERT INTO levels (guild_id, user_id, total_xp)
                 VALUES ($1, $2, $3)
                     ON CONFLICT (guild_id, user_id) DO UPDATE 
                    SET total_xp = levels.total_xp + $3, last_msg = CURRENT_TIMESTAMP
                  WHERE levels.guild_id = $1
                    AND levels.user_id = $2
                    AND EXTRACT(EPOCH FROM (NOW() - levels.last_msg)) > 60
              RETURNING total_xp - $3 AS old_total_xp, total_xp, guild_id, user_id;
"""
```

I try to keep the sql indentation "on a single line" if you understand what I mean here
Update other queries as well pls